### PR TITLE
resolve issue #12: Errors and Fails are hard to read

### PR DESCRIPTION
**Pull request by Fraih001**
I split the incoming string using "\n" then mapped over the array displaying each element. The styling could be improved further.

### CODE REVIEW

File: client/components/Tests/TestExercise.js [hunk 1]
```diff
@@ -234,6 +234,7 @@ export const Editor = (props) => {
       })
       .then((res) => {
         setHasTestPassed(false);
+        console.log(res.data);
```
Comment (bob-zs) [inline client/components/Tests/TestExercise.js:237]: I know this isn't being released to the public, but out of good habit, please delete or at least comment out console logs. I've been told to remove them out of security risks.

**Comment by bob-zs** [comment]
This is definitely a lot nicer to look at, thanks for doing this!
Screen Shot 2023-01-28 at 10 12 57 PM
Screen Shot 2023-01-28 at 10 13 08 PM
Screen Shot 2023-01-28 at 10 13 13 PM

**Comment by bob-zs** [comment]
Thanks @Fraih001 !!!!! Lookin already a lot better!

File: server/api/submitTest.js [hunk 2]
```diff
@@ -28,6 +28,7 @@ router.post('/', async (req, res) => {
     try {
       fs.writeFileSync(filePath, jsCodeWithUserCode);
       const { stderr } = await exec(`npm test ${testFileName}`);
+      // console.log(JSON.stringify(stderr));
```
Comment (bob-zs) [inline server/api/submitTest.js:31]: remove this please

**Review by bob-zs — APPROVED** [review]
👍 great work in formatting. So much more readable.
Please remove those console.logs.
After removing those nits, feel free to merge. Approving it right now, so it doesn't block after removal

**Comment by bob-zs** [comment]
I think i'll start removing the console logs in the frontend.

File: client/components/Tests/TestExercise.js (hunk 1 above)
Comment (bob-zs) [reply client/components/Tests/TestExercise.js:237]: Plus the data is in the networks tab if there is a need to debug it.

**Comment by bob-zs** [comment]
Unless, now I'm thinking....

I can sympathize with having there so we know where to run a `console.log` when we are debugging.

What do you think?
I don't think we should have these running when we open the applications and stuff. I feel the console logs could have less stuff printed to it when we're debugging other parts of the application. It would be helpful if the console was cleaner and didn't have these console logs. 

But having them at least commented out shows where we can debug when we do look at this again. What you think?

**Comment by Fraih001** [comment]
Yeah, I meant to remove most of them, comment out others that I think would be helpful for debugging so I'll do that and add another commit

**Pull request merged**